tests/system/python/e2e/test_e2e_notification_service_with_plugins.py

```diff
@@ -99,8 +102,13 @@ def _add_notification_instance(foglamp_url, payload):
     assert "Notification {} created successfully".format(payload['name']) == jdoc['result']
 
 
-class TestNotificationService:
+def pause_for_x_seconds(x=1):
+    wait_e = Event()
+    wait_e.clear()
+    wait_e.wait(timeout=x)
```

Comment:
@Singhal-Vaibhav  is this better option then time.sleep? 
 it also can be used a fixture at required places?